### DISCUSSION TITLE
Auto Limiting enabling for BigQuery

### DISCUSSION THREAD

**Discussion by JadenisJelu** [getredash/redash#7206] [Ideas]
Hi, I am considering to enable Auto Limiting for BigQuery query runner due to large cache of results crashing the server.

Curious if there are any bottlenecks at the moment that prevents this from being done (perhaps not just to BigQuery but for all query runners) as it seems to only involve a base class change.

**Comment by zachliu** [comment]
not all database engines support metadata like `totalBytesProcessed` or `totalRows`
